### PR TITLE
Add register-login and profile saplings from Grid

### PR DESCRIPTION
**Pull request by eloaverona**
This PR adds the register-login and profiles saplings that are being developed as part of the Grid  UI work. 

### To test:
1. Run:
```
docker-compose -f docker/docker-compose.yaml up --build
```

2. Go to `http://localhost:3030/`. You will be asked to login as an existing user or register a new user:
Screen Shot 2020-04-09 at 9 20 45 AM

3. Once you've register a new user, click the profile icon in the bottom left corner on the menu, which you forward you to `http://localhost:3030/profile`:
Screen Shot 2020-04-09 at 9 27 38 AM

P.S The profile sapling is still under active development so bugs or issues might be present.

### CODE REVIEW

**Review by agunde406 — APPROVED** [review]
tested locally

File: sapling-dev-server/Dockerfile [hunk 1]
```diff
@@ -21,6 +21,20 @@ RUN npm config set unsafe-perm true
 
 COPY . .
 
+# Hack to add register-login and profiles saplings until there's a distribution
+# strategy for saplings
+RUN git clone https://github.com/hyperledger/grid
```
Comment (vaporos) [inline sapling-dev-server/Dockerfile:26]: We should hack this in reverse, so that the official code is here and grid is dependent on splinter-ui.

**Review by dplumb94 — CHANGES_REQUESTED** [review]
I am getting a 404 when trying to add a key in the Profile sapling: 
`splinter-ui-alpha           | 172.25.0.1 - - [09/Apr/2020:19:52:09 +0000] "POST /splinterd/biome/users/7e6b258b-1521-4145-a94a-1e53ddc479ea/keys HTTP/1.1" 404 -`

This may be due to the docker image env vars. You may need to name them according to the node since the SPLINTER_URL variables get built into the image

**Comment by eloaverona** [comment]
@dplumb94 The issue you mentioned is in the profile sapling, and is being fixed in this PR https://github.com/hyperledger/grid/pull/224

**Review by dplumb94 — APPROVED** [review]

**Pull request merged**